modules/Account/app/Providers/Traits/DatabaseSeeders.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Modules\Account\app\Providers\Traits;
+
+trait DatabaseSeeders
+{
+    protected function defineSeeders(): array
```

Comment:
This method can return $this->seeders and when use trait in a class , must define $seeders property .